spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2542,6 +2542,25 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
             None
           }
 
+        case GetArrayStructFields(child, _, ordinal, _, _) =>
```

Comment:
To help other reviewers, here is Spark's definition of `GetArrayStructFields`:

```scala
case class GetArrayStructFields(
    child: Expression,
    field: StructField,
    ordinal: Int,
    numFields: Int,
    containsNull: Boolean)
```

Reply:
Yeah those other three fields aren't relevant for actual execution